packages/linux/package.mk

```diff
@@ -31,6 +31,10 @@ esac
 
 PKG_KERNEL_CFG_FILE=$(kernel_config_path) || die
 
+if [ -n "$($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM crust_config)" ]; then
+    PKG_PATCH_DIRS="$PKG_PATCH_DIRS crust"
+fi
+
 if [ -n "$KERNEL_TOOLCHAIN" ]; then
```

Comment:
Can you please move patches to Allwinner project folder? I would like to avoid polluting packages with project specifics. There are two ways this can be done without too much change:
1. put patches together with others in projects/Allwinner/patches/linux
2. put patches in projects/Allwinner/patches/linux/crust and set LINUX variable in project config to "crust", this is a bit hacky, but a way to have patches separated

Reply:
Moved the patches to projects/Allwinner/patches/linux/crust, turns out that already works with PKG_PATCH_DIRS